app/views/admin/pdfs/index.html.erb

```diff
@@ -1,12 +1,40 @@
-<div class="row">
-  <h1>This is where you can upload pdfs</h1>
-  <br>
-  <%= form_for :pdf, url: admin_pdfs_path, :html => { :multipart => true , class: "form-horizontal" } do |f| %>
-    <div class="form-group">
-      <%= f.label :pdf_file, "Upload PDF", class:"col-sm-2 control-group lead" %>
-      <%= f.file_field :pdf_file %>
-      <br>
-      <%= f.submit "Submit", class: "btn btn-primary col-sm-1" %>
+<h1>This is where you can upload pdfs</h1>
+
+<%= form_for :pdf, url: admin_pdfs_path, :html => { :multipart => true } do |f| %>
+    <%= f.label :pdf, "Upload PDF -->" %>
```

Comment:
Would you mind removing the --> ...I tried to catch these in Jwan's last PR, but maybe I missed this one. Thanks :clap: